src/views/Correction/FmCorrection.vue

```diff
@@ -110,21 +92,14 @@ export default class FmCorrection extends Mixins(CommonMixin, DateMixin, FilingT
       // safety check
       if (!this.correctionFiling) throw (new Error('Missing correction filing'))
 
-      // fetch and store business snapshot
+      // fetch business snapshot
       const businessSnapshot = await this.fetchBusinessSnapshot()
-      this.parseEntitySnapshot(businessSnapshot)
-
-      // parse correction filing into store
-      this.parseCorrectionFiling(this.correctionFiling)
 
-      // get and store ID of filing that is being corrected
-      // NB: we don't care about this but it's needed in the correction filing object
-      const correctedFilingId: number = +this.correctionFiling.correction?.correctedFilingId
-      this.setCorrectedFilingId(correctedFilingId)
+      // parse correction filing and business snapshot into store
+      this.parseCorrectionFiling(this.correctionFiling, businessSnapshot)
```

Comment:
The old code parsed the business snapshot and then parse the correction filing.

Now, similar to the other filings in this UI, the parse method does both, so that things don't clobber each other.